### ISSUE TITLE
Arrow key support for all menu navigation

### ISSUE THREAD

**Issue by robby250** [Quaver/Quaver#892]
Add arrow key + enter navigation support for all the menus in the game. Currently only songlist navigation and pause menu are supported.
What's missing is the Main menu, Options menu, the exit game prompt (it lets you hit enter to confirm and esc to cancel but it could support arrows for highlight selection to make it more intuitive).
Multiplayer could support it too for choosing a lobby to join, maybe even supporting Tab to switch between different controls (switching between choosing a lobby and the buttons at the bottom of the page like "BACK TO MENU", "CREATE MATCH" etc.)

Basically what I'm getting at is that mouse navigation should be optional and not mandatory for a better user experience with multiple input devices.

Also support back/forward mouse side buttons.
